### ISSUE TITLE
Notebook outline: apply yellow color inside the cell as well

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#194118]
Testing #194053

1. Open notebook. Open outline
2. Notice how the yellow rendering is done around the cells.
3. The UI looks bad. The yellow colour should also be applied inside the cell. Same how we do it in the regular editor

![Screenshot 2023-09-26 at 12 13 07](https://github.com/microsoft/vscode/assets/1926584/be8e51f5-0da9-4a87-8344-e01d5faa5637)

**Comment by amunger** [comment]
Not that it looks all that much better, but it should highlight the line if the symbols were gathered correctly
![image](https://github.com/microsoft/vscode/assets/2019016/ed9fbc0f-9447-4838-9451-0520f1d6ba49)

Did you have a kernel selected? (were you able to run a cell?)
Make sure you're on the latest version
Make sure `notebook.gotoSymbols.showAllSymbols` is set

**Comment by isidorn** [comment]
I was not having pylance.
But yes, now it looks a bit worse even with that extra yellow line 😊 

Do we need design help here?

**Comment by amunger** [comment]
yes, I could definitely use some design help. I can also bring it up in the UX sync
